app/code/Magento/Catalog/Test/Mftf/Test/AdminCategoryProductFiltersTest.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCategoryProductFiltersTest">
+        <annotations>
+            <stories value="Product"/>
+            <features value="Catalog"/>
+            <title value="Admin category product filters test"/>
+            <description value="Check if Product Grid filters work correctly on Category page"/>
+            <severity value="MAJOR"/>
+            <group value="catalog"/>
+        </annotations>
+        <before>
+            <!-- Create Category -->
```

Comment:
You don't need to provide comments, as these are displayed just like other steps.
It's enough that `stepKey` for the actual step is descriptive.